common-lib/src/cmr/common/validations/json_schema.clj

```diff
@@ -59,14 +59,20 @@
       (let [message (str/replace (.getMessage e) #"\[Source[^;]+;" "")]
         (errors/throw-service-error :bad-request (str "Invalid JSON: " message))))))
 
+(defn json-string->json-schema
+ "Convert a string to com.github.fge.jsonschema.main.JsonSchema object."
```

Comment:
Why is there only 1 space indent here? Noticing this a lot in the data-center code.